duckduckgo/Android
# ADS: Downloads screen migration

Task/Issue URL: https://app.asana.com/0/0/1203411647299663/f

### Description
Migrate Downloads screen to new components

### Steps to test this PR

- Install from this branch
- Go to Downloads

### UI changes
| Before  | After |
| ------ | ----- |
![Screenshot_20221207_140218_DuckDuckGo](https://user-images.githubusercontent.com/20798495/206199425-9fbdb8a9-3b6e-4796-975d-ed1d0ed603bf.jpg)|![Screenshot_20221207_132951_DuckDuckGo](https://user-images.githubusercontent.com/20798495/206198560-ce6a6c45-16f2-4fb0-9f20-aed5c9e219ce.jpg)|
